### DISCUSSION TITLE
Gradle varient?

### DISCUSSION THREAD

**Discussion by Frooodle** [wiverson/maven-jpackage-template#90] [Ideas]
Any thoughts on a gradle variant for this? Would love to test this for my Stirling-PDF app

**Comment by wiverson** [comment]
https://github.com/Frooodle/Stirling-PDF/ - looks cool!

IIRC there are some Gradle plugins that duplicate some of it, eg adding in jpackage/jlink stuff automatically. It wouldn't be too hard to follow the flow of what this template does and repro it with Gradle. I don't use Gradle (much) tho, so I haven't had much reason/interest in doing a Gradle version.

I have pondered doing a Spring Boot focused version of this template, tho. Either a local web server version with a minimal UI that just opens the user's local browser or maybe leverage the JavaFX WebKit browser for a more desktop (dare I say Electron-ish) version.

The idea with that template would be that it would already have all of the Spring Boot and build stuff ready to go - you could just start adding in your usual Spring Boot stuff and all of the desktop boot-up stuff would be taken care of.

Either way I'm probably going to use Maven tho... ;)
